### PR TITLE
1 file from GHSA sync script run

### PR DESCRIPTION
**Pull request by jasnow**
One file (gems/bootstrap/CVE-2018-14040.yml) from GHSA lib/github_advisory_sync.rb run.
yamllint and "rake" both green.

### CODE REVIEW

File: gems/bootstrap/CVE-2018-14040.yml [hunk 1]
```diff
@@ -10,12 +11,16 @@ description: |
   (CVE-2018-14041), data-container property of tooltip (CVE-2018-14042)
 cvss_v2: 4.3
 cvss_v3: 6.1
+unaffected_versions:
+  - "< 4.0.0"
```
Comment (postmodern) [inline gems/bootstrap/CVE-2018-14040.yml:15]: Are we really sure that versions before `4.0.0` are not affected? I can't seem to find information that confirms this.

**Review by postmodern — CHANGES_REQUESTED** [review]
I cannot find any information that confirms that versions before `4.0.0` are not affected. Neither the GHSA nor the NVD links mentions this.

**Comment by postmodern** [comment]
Correction, GHSA claims only versions `>= 4.0.0, < 4.1.2` are affected. The blog post, NVD link, and other mailing list posts don't seem to mention this?

File: gems/bootstrap/CVE-2018-14040.yml (hunk 1 above)
Comment (jasnow) [reply gems/bootstrap/CVE-2018-14040.yml:15]: Removed

**Review by postmodern — APPROVED** [review]

**Pull request merged**